packages/overture-schema-divisions-theme/src/overture/schema/divisions/division_area/models.py

```diff
@@ -0,0 +1,89 @@
+"""Division area models for Overture Maps divisions theme."""
+
+from typing import Annotated, Literal
+
+from pydantic import ConfigDict, Field
+
+from overture.schema.core import (
+    OvertureFeature,
+)
+from overture.schema.core.geometry import Geometry, GeometryTypeConstraint
+from overture.schema.core.models import (
+    Named,
+    Names,
+)
+from overture.schema.core.types import CountryCode, Id, RegionCode
+from overture.schema.divisions.division_area.enums import AreaBoundaryClass
+from overture.schema.validation import (
+    exactly_one_of,
+)
+
+from ..enums import PlaceType
+
+
+@exactly_one_of("is_land", "is_territorial")
+class DivisionArea(OvertureFeature, Named):
+    """Division areas are polygons that represent the land or maritime area covered by a division.
+
+    Each division area belongs to a division which it references by ID, and for which the division
+    area provides an area polygon. For ease of use, every division area repeats the subtype, names,
+    country, and region properties of the division it belongs to.
+    """
+
+    model_config = ConfigDict(title="division_area")
+
+    # Core
+
+    theme: Literal["divisions"]
+    type: Literal["division_area"]
+    geometry: Annotated[
+        Geometry,
+        GeometryTypeConstraint("Polygon", "MultiPolygon"),
+        Field(
+            description="Division area geometries MUST be polygons or multi-polygons as defined by the GeoJSON schema.",
+        ),
+    ]
+
+    # Required
+
+    names: Names
+    subtype: PlaceType
+    class_: Annotated[
+        AreaBoundaryClass,
+        Field(alias="class"),
+    ]
+    is_land: Annotated[
+        bool | None,
+        Field(
+            description="""A boolean to indicate whether or not the feature geometry represents the land-clipped, non-maritime boundary. The geometry can be used for map rendering, cartographic display, and similar purposes.""",
+            strict=True,
+        ),
+    ] = None
+    is_territorial: Annotated[
+        bool | None,
+        Field(
+            description="""A boolean to indicate whether or not the feature geometry represents Overture's best approximation of this place's maritime boundary. For coastal places, this would tend to include the water area. The geometry can be used for data processing, reverse-geocoding, and similar purposes.""",
+            strict=True,
+        ),
+    ] = None
```

Comment:
This is a comment on the schema today, not the Pydantic port, but I'm struggling to remember and/or rationalize how we have both `class` and `is_land`/`is_territorial`. ...

Reply:
Difficulties modeling the validation rules for `is_land` / `is_territorial` are another smell.